keyboards/melgeek/z70ultra/rev1/rev1.c

```diff
@@ -0,0 +1,171 @@
+/* Copyright 2020 MelGeek <melgeek001365@gmail.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "rev1.h"
+
+
+#ifdef RGB_MATRIX_ENABLE
+__attribute__ ((weak))
```

Comment:
Is there a reason for setting this weak?

Reply:
I refer to keyboards/keebio/iris.

Reply:
```suggestion
```